crm_batch_leads/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (c) 2010-2012 Elico Corp. All Rights Reserved.
+#    Author: Yannick Gouin <yannick.gouin@elico-corp.com>
+#    Author: LIN Yu <lin.yu@elico-corp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'CRM & Projects ACL',
+    'version': '1.1',
+    'category': 'Generic Modules/Others',
+    'complexity': 'easy',
+    'author': 'Elico Corp,Odoo Community Association (OCA)',
```

Comment:
@elicoidal  so ,when i push to elido odoo,I need keep the author with OCA . and when we push we will adapt the README again?